src/app/utils/functions/dateFunctions.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable radix */
 import moment, { Moment } from 'moment';
+import { DEFAULT_DATE_FORMAT } from '../../../global/constants';
 
 const defaultLocale = 'NL';
 const systemDateFormat = 'YYYY-MM-DD';
```

Comment:
do we still need systemDateFormat if we have DEFAULT_DATE_FORMAT in the constants

Reply:
Grrrr, no :-)